setup.py

```diff
@@ -33,8 +33,9 @@ def _get_plugin_version_dict():
     _version_path = os.path.join(this_directory, "dbt", "adapters", "snowflake", "__version__.py")
     _semver = r"""(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)"""
     _pre = r"""((?P<prekind>a|b|rc)(?P<pre>\d+))?"""
+    _build = r"""(\+build[0-9]+)?"""
```

Comment:
The `?` renders it optional which is what I think we want -- there will be many builds without this suffix.